lib/screens/surveydetails/widget/answer_text_field_widget.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:lydiaryanfluttersurvey/model/ui/question_ui_model.dart';
+import 'package:lydiaryanfluttersurvey/resources/dimensions.dart';
+import 'package:lydiaryanfluttersurvey/screens/widgets/app_input_widget.dart';
+
+class AnswerTextFieldWidget extends StatefulWidget {
+  final QuestionUiModel question;
+  final Function(List<String>) onAnswered;
+
+  const AnswerTextFieldWidget({
+    Key? key,
+    required this.question,
+    required this.onAnswered,
+  }) : super(key: key);
+
+  @override
+  State<AnswerTextFieldWidget> createState() => _AnswerTextFieldWidgetState();
+}
+
+class _AnswerTextFieldWidgetState extends State<AnswerTextFieldWidget> {
+  final List<String> _answers = [];
+  final List<TextEditingController> _controllers = [];
+
+  void _onTextChanged(
+    int index,
+    String text,
+  ) {
+    _answers[index] = text;
+    widget.onAnswered(_answers);
+  }
+
+  @override
+  void dispose() {
+    for (var element in _controllers) {
+      element.dispose();
+    }
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    TextField(
+      onChanged: (text) => _onTextChanged(-1, text),
+      controller: TextEditingController(),
+    );
```

Comment:
@ryan-conway not sure why you create a widget here without any usage?

Reply:
My mistake, I was checking how `TextField` works and forgot to remove it 🙈 Updated in [4b0cdcf](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/89/commits/4b0cdcf90b21f67e9c862b40df27636b5fe94f60)